deb/jjs-oneclick

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+if [ "$(whoami)" != root ]
+then
+    echo 'jjs-oneclick must be run as root!'
+    exit 1
+fi
+which psql >/dev/null || exit $?
+
+echo '# Setting up config'
+su -c 'jjs-setup --data-dir /var/lib/jjs --install-dir /usr/share/jjs --setup-config' -s "$SHELL" jjs
+
+echo '# Setting up problems'
```

Comment:
Use jjs-setup instead

Reply:
Well, for this case you can add TODO instead because setup is unable to partially update DB